paymentsheet/src/main/java/com/stripe/android/customersheet/data/CustomerPaymentOption.kt

```diff
@@ -0,0 +1,11 @@
+package com.stripe.android.customersheet.data
+
+internal sealed interface CustomerPaymentOption {
+    data object GooglePay : CustomerPaymentOption
+
+    data object Link : CustomerPaymentOption
+
+    data class Saved(val id: String) : CustomerPaymentOption
+
+    data object None : CustomerPaymentOption
+}
```

Comment:
Wondering if `SavedSelection` should be used instead since they are effectively the same type.

Reply:
Seems reasonable to me. But I don't feel strongly one way or another.